packages/psss-server/src/connections/EntityConnection.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Tupaia
+ * Copyright (c) 2017 - 2021 Beyond Essential Systems Pty Ltd
+ */
+
+import { PSSS_ENTITY, PSSS_HIERARCHY } from '../constants';
+import { ApiConnection } from './ApiConnection';
+
+const { ENTITY_URL = 'http://localhost:8050/v1' } = process.env;
+
+type EntityObject = {
+  id: string;
+  code: string;
+  name: string;
+};
+
+interface EntityFilter {
+  type?: string | string[];
+}
+
+const buildFilter = (fields: EntityFilter) =>
+  Object.entries(fields)
+    .map(([key, value]) => `${key}:${value}`)
+    .join(';');
```

Comment:
@rohan-bes similarly here: this repeats logic existing in `entity-server`, but I didn't want to add that package as a dependency of `psss-server`